src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -87,13 +87,13 @@ const styles = StyleSheet.create({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[400],
     textAlign: 'left',
-    color: isHeaderHide ? COLORS.white : COLORS.blackgrey
+    color: isHeaderHide ? COLORS.white2 : COLORS.blackgrey
   }),
   seeMemberText: (followType) => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
+    color: followType === 'incognito' ? COLORS.anon_primary : COLORS.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The colors for `color` and `seeMemberText` have been updated. Make sure these new colors (`COLORS.white2`, `COLORS.anon_primary`) are defined in your color palette and they provide the desired visual effect on different screens and themes (light/dark mode).
```diff
- color: isHeaderHide ? COLORS.white : COLORS.blackgrey
+ color: isHeaderHide ? COLORS.white2 : COLORS.blackgrey

- color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
+ color: followType === 'incognito' ? COLORS.anon_primary : COLORS.blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->